.github/workflows/build-src.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: build-src
+
+on: [push, pull_request, workflow_dispatch]
+
+concurrency: ci-${{ github.ref }}
+
+jobs:
+  BuildHamiCoreLib:
+    runs-on: ubuntu-latest
+    timeout-minutes: 1440
+    strategy:
+      fail-fast: true
+    name: Build libvgpu
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
```

Comment:
```suggestion
      -
        name: Set up QEMU
        uses: docker/setup-qemu-action@v3 
      - name: Set up Docker Buildx
        id: buildx
        uses: docker/setup-buildx-action@v3
```

Reply:
Add linux/amd64,linux/arm64 muti-arch

Reply:
purpose to add this only for check if pr has any build error?   in hami  repo  will include tihis libvgpu.so compile